gem/migrations/0014_convert_recomended_articles.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def create_recomended_articles(main_article, article_list):
+    '''
+    Creates recommended article objects from article_list
+    and _prepends_ to existing recommended articles.
+    '''
+    from molo.core.models import ArticlePageRecommendedSections
```

Comment:
It's improper to import a model directly in migrations. 
Use `apps.get_model("yourappname", "Model")` instead.
```python
ArticlePageRecommendedSections = apps.get_model("core", "ArticlePageRecommendedSections")
```